assets/schema_input.json

```diff
@@ -16,8 +16,8 @@
             "contigs": {
                 "type": "string",
                 "format": "file-path",
-                "pattern": "^\\S+\\.f(ast|n)?a\\.gz$",
-                "errorMessage": "FASTA file containing assembled contigs, cannot contain spaces and must have extension '.fa.gz' or '.fasta.gz'"
+                "pattern": "^\\S+\\.f(ast|n)?a(\\.gz)?$",
+                "errorMessage": "FASTA file containing assembled contigs, cannot contain spaces and must have extension ('.fa', '.fasta', or '.fna'). Compressed files must be ('.gz')"
```

Comment:
Sorry, I'm nit-picking, but could the last part instead read:

"GZIP-compressed files must additionally have the '.gz' extension ."

Reply:
I agree! I got sheepish as the error message got long, and I tried to too neatly wrap it up. The change does make it more clear. ef2793365765564a0e602703d7740db8a69a1e87